docs/features/software-templates/installation.md

```diff
@@ -0,0 +1,185 @@
+
+  const filePreparer = new FilePreparer();
+  const githubPreparer = new GithubPreparer();
+  const preparers = new Preparers();
+
+  // Register default preparers
+  preparers.register('file', filePreparer);
+  preparers.register('github', githubPreparer);
+
+  // Create Github client with your access token from environment variables
+  const githubClient = new Octokit({ auth: process.env.GITHUB_ACCESS_TOKEN });
+  const publisher = new GithubPublisher({ client: githubClient });
+
+  const dockerClient = new Docker();
+  return await createRouter({
+    preparers,
+    templaters,
+    publisher,
+    logger,
+    dockerClient,
+  });
+}
+```
+
+Once the `scaffolder.ts` router setup file is in place, add the router to
+`packages/backend/src/index.ts`:
+
+```ts
+import scaffolder from './plugins/scaffolder';
+
+const scaffolderEnv = useHotMemoize(module, () => createEnv('scaffolder'));
+
+const service = createServiceBuilder(module)
+  .loadConfig(configReader)
+  /** several different routers */
+  .addRouter('/scaffolder', await scaffolder(scaffolderEnv));
+```
+
+### Adding Templates
+
+At this point the scaffolder backend is installed in your backend package, but
+you will not have any templates available to use. These need to be added to the
+software catalog, as they are represented as entities of kind
+[Template](/docs/features/software-catalog/descriptor-format.md#kind-template).
+You can find out more about adding templates [here](./adding-templates.md).
+
+To get up and running and try out some templates quickly, you can add some of
+our example templates through static configuration. Add the following to the
+`catalog.locations` section in your `app-config.yaml`:
+
+```yaml
+catalog:
+  locations:
+    # Backstage Example Templates
+    type: github
+    target: https://github.com/spotify/backstage/blob/master/plugins/scaffolder-backend/sample-templates/react-ssr-template/template.yaml
+    type: github
+    target: https://github.com/spotify/backstage/blob/master/plugins/scaffolder-backend/sample-templates/springboot-grpc-template/template.yaml
+    type: github
+    target: https://github.com/spotify/backstage/blob/master/plugins/scaffolder-backend/sample-templates/create-react-app/template.yaml
+    type: github
+    target: https://github.com/spotify/cookiecutter-golang/blob/master/template.yaml
+```
+
+### Runtime Dependencies
+
+For the scaffolder backend plugin to function, it needs a GitHub access token,
+and access to a running Docker daemon. You can create a GitHub access token
+[here](https://github.com/settings/tokens/new), select `repo` scope only. Full
+docs on creating private GitHub access tokens is available
+[here](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token).
+Note that the need for private GitHub access tokens will be replaced with GitHub
+Apps integration further down the line.
+
+The GitHub access token is passed along using the `GITHUB_ACCESS_TOKEN`
```

Comment:
Here is nice an alert about the [org trick](https://github.com/spotify/backstage/issues/1881#issuecomment-671355877)